.hashibot.hcl

```diff
@@ -7,8 +7,36 @@ behavior "remove_labels_on_reply" "remove_stale" {
     labels = ["waiting-reply"]
 }
 
+poll "stale_issue_closer" "waiting_reply_closer" {
+    schedule = "1 0 * * *"
+    no_reply_in_last = "480h" # 20 days
+    max_issues = 50
+    sleep_between_issues = "5s"
+    labels = ["waiting-reply-🤖"]
+    message = <<-EOF
+    Hey there,
+
+    This issue has been automatically closed because there hasn't been any activity for at least _20_ days.
+
+    If you are still experiencing problems, or still have questions, feel free to [open a new one](https://github.com/hashicorp/consul/issues/new) :+1:.
+    EOF
+}
+
+poll "stale_issue_closer" "old_issue_closer" {
+    schedule = "*/5 * * * *"
+    no_reply_in_last = "1s" # hack to close issue with that label immediately.
+    max_issues = 50
+    sleep_between_issues = "5s"
+    labels = ["close-old-issue-🤖"]
+    message = <<-EOF
+    Hey there,
+
+    This issue was reported on a version of Consul that is quite old. Based on the volume of changes since that version we encourage you to try reproducing with the latest version of Consul. If you feel the issue is critical and on an older version of Consul, or still exists in the current version, feel free to [open a new issue](https://github.com/hashicorp/consul/issues/new) :+1:.
+    EOF
+}
+
 # poll "closed_issue_locker" "locker" {
-#   schedule             = "0 50 1 * * *"
+#   schedule             = "1 1 * * *"
```

Comment:
The scheduling was left to an older version of this library: https://github.com/robfig/cron . It had some extra level of resolution supporting seconds... I think the new version of that lib gets rid of this due to all the confusion it has caused. Leave the leftmost position set to `0`. As configured this would run every hour at the first minute and second.